.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"parser": "typescript",
+	"printWidth": 120,
+	"trailingComma": "none",
+	"tabWidth": 4,
+	"singleQuote": true,
```

Comment:
do we have to use single quote? feels like it's going to be a big change....
plus: https://stackoverflow.com/a/25453898

Reply:
Umm, this is not that of a hard constraint and I actually don't manage to change the quotes, the extension actually reformats the code itself. If you observe `Login.jsx`, all the imports and normal strings get wrapped in single quotes and all the JSX elements will have double quotes. However, this is just experimental to have some form of consistency, and we can remove it totally. I myself don't follow which is why I have prettier manage that for me :p